mev

```diff
@@ -43,7 +43,7 @@ case "$1" in
         redis
     ;;
   listener)
-        kubectl exec -ti deploy/mev-inspect -- ./listener $2
+        kubectl exec -ti deploy/mev-inspect -- ./listener $2 $3
```

Comment:
Having to pass the RPC type to the user will become burdensome on every command

Reply:
I agree, will commit new changes